eng/helix/vstest/runtests.cmd

```diff
@@ -23,6 +23,12 @@ if %errorlevel% equ 0 (
     exit 1
 )
 
-%DOTNET_ROOT%\dotnet vstest %target% --logger:trx --logger:console;verbosity=normal
+%DOTNET_ROOT%\dotnet vstest %target% --logger:trx;LogFileName=test-results.xml --logger:console;verbosity=normal
 
+set _ERRORLEVEL=%ERRORLEVEL%
 
+rem Normalize test result file name
+
+ren TestResults\test-results*.xml test-results.xml
```

Comment:
Is this still needed with the `LogFileName=test-results.xml` change just above?

Reply:
Yes, vstest inserts timestamp into `test-results.xml`